include/aspect/gravity_model/function.h

```diff
@@ -85,6 +85,12 @@ namespace aspect
          * A function object representing the gravity.
          */
         Functions::ParsedFunction<dim> function;
+
+        /**
+         * The coordinate representation to evaluate the function. Possible
+         * choices are depth, cartesian and spherical.
+         */
+        Utilities::Coordinates::CoordinateSystem coordinate_system;
```

Comment:
I think you need to add `#include <aspect/utilities.h>` so that ASPECT knows what a coordinate system is. Talk to me tomorrow if you want to know why it is not a problem on your laptop, but it is for the tester.